app/views/shared/check_answers/_bank_statements.html.erb

```diff
@@ -1,12 +1,14 @@
 <% read_only = false unless local_assigns.key?(:read_only) %>
 
-<% url = if partner
-           providers_legal_aid_application_partners_bank_statements_path(@legal_aid_application)
-         else
-           providers_legal_aid_application_bank_statements_path(@legal_aid_application)
-         end %>
-
-<% individual = partner ? "Partner" : "Client" %>
+<% if partner
+     individual = "Partner"
```

Comment:
Could we set each of these on the controller(s - i guess for each cya page) rather than in each of the partials?

Reply:
@RoseSAK Hmm yeah maybe you're right. It just feels like there's a lot of logic in the view and I wondered whether it could be moved elsewhere to clean it up a bit and avoid writing very similar stuff in each view, such as:

```
if partner
  individual = "partner"
  individual_with_determiner = t("generic.partner")
else
 individual = "Client"
 individual_with_determiner = t("generic.client")
end
```

Not a blocking comment or a major issue though

Reply:
Yeah that would be good, I'll merge this and write a separate ticket for that